mlflow/tracking/_tracking_service/utils.py

```diff
@@ -191,18 +191,27 @@ def _get_databricks_uc_rest_store(store_uri, **_):
 
 
 _tracking_store_registry = TrackingStoreRegistry()
-_tracking_store_registry.register("", _get_file_store)
-_tracking_store_registry.register("file", _get_file_store)
-_tracking_store_registry.register("databricks", _get_databricks_rest_store)
-_tracking_store_registry.register(_DATABRICKS_UNITY_CATALOG_SCHEME, _get_databricks_uc_rest_store)
 
-for scheme in ["http", "https"]:
-    _tracking_store_registry.register(scheme, _get_rest_store)
 
-for scheme in DATABASE_ENGINES:
-    _tracking_store_registry.register(scheme, _get_sqlalchemy_store)
+def _register_tracking_stores():
+    """Register tracking stores.
+    This method is to be called upon the module initialization or per user's request to refresh"""
```

Comment:
Do we need this docstring?

Reply:
We normally don't expect users to call this function because it's in a private module.

Reply:
My style is verbose. 

I am happy to remove if you think it is self-descriptive enough?

Reply:
Yeah, I think it's  self-descriptive enough.

Reply:
changed in 4e4e47254cb86e6b4cba60b3a019e89f250b0ad7